config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root 'users#index'
+  get 'users/:id', to: 'users#show'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get 'users/:id/posts', to: 'posts#index'
+  get 'users/:id/posts/:post_id', to: 'posts#show'
```

Comment:
- [ ] Great work so far but it would be nice if you make your routes using resources to follow the latest syntax of rails which is more maintainable 👍 which you can see below ⬇️

```rb

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end


```

Reply:
@HAMMAS-SALEEM Thanks for your suggestion, but this is not part of the requirements, this could be optional, but I'll do it.

Reply:
Dear @CesarHerr, actually when you are making this project it is not mentioned in the project requirements which way you should use but whenever you make code then technically you always use the latest syntax so in my opinion it is a basic thing dear 😊 but if you think it might be unfair to make this requirement mandatory then you can ask for a second opinion to settle this discussion 👍